logger/configurer/configurer.go

```diff
@@ -0,0 +1,137 @@
+package configurer
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/coreos/go-etcd/etcd"
```

Comment:
Ya.  This is not the only component that's still using that.  If we want to get off of it, we should do it across the board as a chore.